src/game/camera.c

```diff
@@ -4545,15 +4675,21 @@ void play_camera_buzz_if_c_sideways(void) {
 }
 
 void play_sound_cbutton_up(void) {
+#ifndef REONUCAM
     play_sound(SOUND_MENU_CAMERA_ZOOM_IN, gGlobalSoundSource);
+#endif
 }
 
 void play_sound_cbutton_down(void) {
+#ifndef REONUCAM
```

Comment:
cleaner (both codewise and optimization wise) to ifdef this whole block and then if reonucam is defined instead do `#define play_sound_cbutton_up()`